index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <!-- Given a descriptive title with a icon-->
+    <link
+      rel="shortcut icon"
+      type="image/png"
+      href="./icon-image/Horiseon-icon.png"
+    />
+    <title>Horiseon:Marketing Experts For You</title>
+  </head>
+
+  <body>
+    <!-- Header section with the nav bar containing links -->
+
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!-- Navigation bar with the website links-->
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- Main is divided into three sections with the img and description   -->
+
+    <div class="hero"></div>
+
+    <!-- Added a id "search-engine-optimization" to fix the nav link -->
+
+    <main class="content">
+      <section>
```

Comment:
Instead of wrapping the `div` in a section, consider just changing the `div` to section. It makes the structure simpler and we don't really need that extra wrapper layer.